pkg/resolver/search.go

```diff
@@ -295,10 +281,57 @@ func formatDataMap(data map[string]interface{}) map[string]interface{} {
 			item[key] = strconv.FormatInt(int64(v), 10)
 		case map[string]interface{}:
 			item[key] = formatLabels(v)
+		case []interface{}:
+			item[key] = formatArray(v)
 		default:
 			klog.Warningf("Error formatting property with key: %+v  type: %+v\n", key, reflect.TypeOf(v))
 			continue
 		}
 	}
 	return item
 }
+
+func WhereClauseFilter(input *model.SearchInput) []exp.Expression {
+	var colDs exp.IdentifierExpression
```

Comment:
Would it possible to combine the `goqu.Ex{}` syntax with `jsonb` ?  It could make this function much simpler.
Here's the documentation for the syntax I'm thinking: http://doug-martin.github.io/goqu/docs/selecting.html#where

The query will look something like this (without considering jsonb)
```go
.Where(goqu.Ex{
  "kind": string[]{"pod", "deployment"},
  "name": string[]{"name-a", "name-b"},
})
```

Reply:
I haven't found how to combine json and `goqu.Ex`. But, simplified the whereClause function with `goqu.L` for json columns.